index.html

```diff
@@ -165,69 +197,78 @@ <h1>About me</h1>
         </p>
         <div class="about-me-btn"><input type="button" class="about-me-button" value="Get my Resume"></div>
       </div>
-      <div class="about-me1-separation">
-        <div class="line2"></div>
-      </div>
+      <div class="about-me-background"></div>
+    </div>
+    <div class="about-me1-separation">
+      <div class="line2"></div>
     </div>
     <div class="about-me-2">
-      <div class="topic">
-        <img src="./svg/language.svg" alt="" />
-        <p>Language</p>
-      </div>
-      <ul class="discussion">
-        <li><a href="https://en.wikipedia.org/wiki/JavaScript">Javascript</a></li>
-        <li><a href="https://en.wikipedia.org/wiki/Ruby_(programming_language)">Ruby</a></li>
-        <li><a href="https://en.wikipedia.org/wiki/HTML">html</a></li>
-        <li><a href="https://en.wikipedia.org/wiki/CSS">css</a></li>
-      </ul>
-      <div class="topic">
-        <img src="./svg/frameworks.svg" alt="" />
-        <p>Frameworks</p>
+      <div class="grid1">
+        <div class="topic">
+          <img src="./svg/language.svg" alt="" />
+          <p>Language</p>
+        </div>
+        <ul class="discussion">
```

Comment:
- [ ] Please apply the correct gap between the technologies, each technology should be listed in the same line and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/161121236-3edf6424-cad3-4a83-b6fe-77b46a118d3e.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/161121281-a01a1912-cbfd-44d5-bf94-473bcfaf936a.png)